geoalchemy2/comparator.py

```diff
@@ -86,7 +86,7 @@ def __getattr__(self, name):
         # This is not to mess up with SQLAlchemy's use of
         # hasattr/getattr on Column objects.
 
-        if not name.startswith('ST_'):
+        if not name.lower().startswith('st_'):
```

Comment:
With https://github.com/sqlalchemy/sqlalchemy/commit/702c96b2ee94a394ab1b4d528e3090c4d3c42b91 this change may still be useful. @adrien-berchet interested to have your advice about all this.

Reply:
Indeed, I think this is the only change we need in geoalchemy2. I will test that.